provider/ansible/request.rb

```diff
@@ -94,7 +96,12 @@ def response_output(prop, hash_name)
           "#{hash_name}.get(#{unicode_string(prop.name)})"
         end
       end
+      # rubocop:enable Metrics/MethodLength
 
+      # rubocop:disable Metrics/MethodLength
+      # rubocop:disable Metrics/AbcSize
+      # rubocop:disable Metrics/CyclomaticComplexity
+      # rubocop:disable Metrics/PerceivedComplexity
       def request_output(prop, hash_name)
```

Comment:
Agree w/ Rubocop here. This method is a great candidate for refactoring. Please simplify it.